docker/dvc_operator.Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.6.7-slim
```

Comment:
In prod I'd use whatever the newest Python version supported by both `paidsearch` and `bidding` does. We'll probably control this through the `makefile` or a script.

Side question: What do you think of using Python over Bash for such scripts? Bash has much better primitives for doing "shell-y" stuff (pipes, subshells, other stuff like that). I think I favor Bash but I don't feel strongly.